src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
```

Comment:
- [ ] As I mentioned in the comment above,  Kindly make sure you fix your linters errors from your code base  instead of disabling them, this for clean code, best practices reasons and as required for all projects.